spaces/S000174/README.md

```diff
@@ -0,0 +1,11 @@
+---
+uid: S000174
+slug: pol-space
+name: Pol's space
+refs:
+  - doi: 10.4064/FM-97-2-53-55
+    name: "A perfectly normal locally metrizable non-paracompact space"
+---
+
+Constructed by Pol in {{doi:10.4064/FM-97-2-53-55}} to obtain an example of a
```

Comment:
"Pol" --> "R. Pol" to distinguish from E. Pol who also has various papers in topology from the same time.